man/ocamlopt.1

```diff
@@ -716,6 +716,33 @@ as a file name, even if it starts with a dash (-) character.
 .BR \-help \ or \ \-\-help
 Display a short usage summary and exit.
 
+.SH OPTIONS FOR THE FLAMBDA MIDDLE-END
+
+When the Flambda code generator has been enabled at configuration time,
+its behavior may be tuned up with the following additional options:
+.TP
+.B \-02
+Perform more optimisation than usual. Compilation times may be lengthened.
+.TP
+.B \-03
+Perform even more optimisation than usual, possibly including unrolling
+of recursive functions. Compilation times may be significantly lengthened.
+.TP
+.B \-Oclassic
+Makes inlining decisions at the point of definition of a function rather
+than at the call site(s). This mirrors the behaviour of OCaml compilers
+not using Flambda. Compared to compilation using the new Flambda inlining
+heuristics (for example at -O2) it produces smaller .cmx files, shorter
+compilation times and code that probably runs rather slower. When using
+-Oclassic, only the following options described in this section are
+relevant: -inlining-report and -inline. If any other of the options
+described in this section are used, the behaviour is undefined and may
+cause an error in future versions of the compiler.
```

Comment:
Sorry I missed that point earlier, but this last paragraph
```
When using -Oclassic, only the following options ...
... 
cause an error in future versions of the compiler.
```
refers to the description in the manual, which has not been imported to the man page. It is thus better to remove this paragraph (for now).

(As a tip, you don't need to merge the last version of trunk in your branch when you update a PR if there are no conflicts).